src/App.tsx

```diff
@@ -1,10 +1,16 @@
 import "./App.css";
+import Practice1 from "./components/Practice1";
 
 function App() {
 	return (
-		<header className="bg-[#94A3B8] text-center p-[20px] text-4xl text-[#F9FAFB]">
+		<div>
```

Comment:
I think we could remove this outer div without any visual changes. I could be wrong though, could you confirm that?

Reply:
Yes removed it now.